src/wranglerArtifactManager.ts

```diff
@@ -0,0 +1,80 @@
+import { open, readdir } from "fs/promises";
+
+interface OutputEntryBase<T extends string> {
+	version: number;
+	type: T;
+}
+
+export interface OutputEntryPagesDeployment
```

Comment:
Can we use zod here?

Reply:
Ah! This was a comment a I mean to leave. We aren't using zod anywhere else in this repo, so I wasn't sure if that would be appropriate. You think we should start using it?

Reply:
I think so! It would be really great for catching if the schema changes at all for some bizarre reason

Reply:
Cool. Will do